.gitmodules

```diff
@@ -31,12 +25,18 @@
 [submodule "src/lib/ecl"]
 	path = src/lib/ecl
 	url = https://github.com/PX4/ecl.git
-[submodule "cmake/cmake_hexagon"]
-	path = cmake/cmake_hexagon
-	url = https://github.com/ATLFlight/cmake_hexagon
 [submodule "src/drivers/gps/devices"]
 	path = src/drivers/gps/devices
 	url = https://github.com/PX4/GpsDrivers.git
 [submodule "src/lib/micro-CDR"]
 	path = src/lib/micro-CDR
 	url = https://github.com/eProsima/micro-CDR.git
+[submodule "platforms/nuttx/src/modules/uavcan/libuavcan"]
```

Comment:
@dagar WRT flattening proposals, the Makefile rules to allow tab completion of build configs using the ${PX4_SOURCE_DIR}/platforms/${OS}/${BOARD}/${CONFIG}.cmake is much harder. I'd rather tackle that at a later date if possible. It nicer but not blocking the objective of making it modular.